libs/nft-contract/src/contract.ts

```diff
@@ -9,11 +9,14 @@ const getAbi = () => {
 
 const getContractAddress = () => {
   console.log(`Loading NFT contract for: ${process.env.CONFIG_NAME}`);
-  const deployment = require(`./deployed/${process.env.CONFIG_NAME}.json`);
-  if (!deployment) {
+  try {
+    const deployment = require(`./deployed/${process.env.CONFIG_NAME}.json`);
+    console.log('Loaded succesfully');
+    return deployment.contract;
+  } catch (err) {
+    console.error(`We failed to load ./deployed/${process.env.CONFIG_NAME}.json`)
```

Comment:
I should remove these console.log statements, (or replace with log.xxx)

Reply:
Indeed